programs/utilities/renderdistance.sc

```diff
@@ -0,0 +1,105 @@
+// Displays the current render distance for servers with a dynamic render distance mod. (Like the one made by Henkelmax)
+// When a player joins they can see the render distance at the bottom of the player list.
+// It refreshes every 1200 ticks (1 minute) and color codes the distance according to carpet TPS logging colors.
+// The refresh time of the display and the min/max render distance can be altered with commands. This will be saved.
+// Made by Sulpherstær with aid from people in the carpet discord. (notably: Surf, altrisi, rv3r, Firigion, and Crec0)
+
+global_refresh_time_in_ticks = 1200;
+global_server_max_render_distance = 32;
+global_server_min_render_distance = 10;
+
+__config() -> {

```

Comment:
If you want to be able to use this script in conjunction with another mod which adds the `/renderdistance` command, you should add `'allow_command_conflicts'->true` to the config or it may not work

Reply:
At the moment there does not appear to be a conflict at all. But I'll add the line still. 👍